File: Week6/count_test.py
import time  # Import the time module

def main():
    n = 0

    # Start timing
    start_time = time.time()

    # Your original loop
    while n < 1000000000:
        n += 1

    # Stop timing
    end_time = time.time()

    # Calculate the elapsed time
    elapsed_time = end_time - start_time

    # Output the elapsed time
    print(f"Elapsed time: {elapsed_time:.6f} seconds")
    print("Done!")

if __name__ == "__main__":
    main()

File: Week6/count_test.cpp
#include <iostream>
#include <chrono>  // Include the chrono library

using namespace std;

int main() {
    int n = 0;

    // Start timing
    auto start = chrono::high_resolution_clock::now();

    // Your original loop
    while (n < 1000000000) {
        n++;
    }

    // Stop timing
    auto end = chrono::high_resolution_clock::now();

    // Calculate the elapsed time
    chrono::duration<double> elapsed = end - start;

    // Output the elapsed time
    cout << "Elapsed time: " << elapsed.count() << " seconds" << endl;

    return 0;
}
